backend/src/model/response.rs

```diff
@@ -2,10 +2,19 @@
 
 use serde::{Deserialize, Serialize};
 use typeshare::typeshare;
+use utoipa::ToSchema;
+
+use super::dto::{PlantsDto, SeedDto};
 
 /// The default return type of the server.
 #[typeshare]
-#[derive(Debug, Serialize, Deserialize)]
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
+#[aliases(
+    BodyVecSeed = Body<VecSeed>, VecSeed = Vec<SeedDto>, // TODO: not rendered as array in swagger
```

Comment:
That is a problem with generics (it's one of the reasons why I want to remove the `Body` struct).
So no what is currently rendered in swagger-ui is actually not entirely correct (that's why the TODO).
Maybe there is some other way to do this, but utoipa is somewhat new and there are no examples on using nested generics. After some digging the best I found was https://docs.rs/utoipa/3.1.0/utoipa/derive.ToSchema.html#generic-schemas-with-aliases.

Reply:
Please report this problem upstream. If it is a new project and has some obvious bugs it is imho okay, if they at least fix such problems quickly. It is only by chance that in this case we could remove the `Body`  struct. In other cases, this might not be possible.

Reply:
See https://github.com/juhaku/utoipa/issues/524